showcase/tests/integration/components/hds/app-side-nav/index-test.js

```diff
@@ -254,29 +257,81 @@ module('Integration | Component | hds/app-side-nav/index', function (hooks) {
   @onDesktopViewportChange={{this.onDesktopViewportChange}}
 >
   <span id='test-app-side-nav-body' />
-  <span class='hds-app-side-nav-hide-when-minimized' />
 </Hds::AppSideNav>`);
 
     await click('.hds-app-side-nav__toggle-button');
-    assert.dom('.hds-app-side-nav-hide-when-minimized').hasAttribute('inert');
+    assert.dom('.hds-app-side-nav__wrapper-body').hasAttribute('inert');
 
     await this.changeBrowserSize(false);
     assert.deepEqual(
       calls[1],
       [false],
       'resizing to mobile triggers a false event'
     );
-    assert.dom('.hds-app-side-nav-hide-when-minimized').hasAttribute('inert');
+    assert.dom('.hds-app-side-nav__wrapper-body').hasAttribute('inert');
 
     await this.changeBrowserSize(true);
     assert.deepEqual(
       calls[2],
       [true],
       'resizing to desktop triggers a true event'
     );
-    assert
-      .dom('.hds-app-side-nav-hide-when-minimized')
-      .doesNotHaveAttribute('inert');
+    assert.dom('.hds-app-side-nav__wrapper-body').doesNotHaveAttribute('inert');
+    assert.dom('body', document).doesNotHaveStyle('overflow');
+  });
+
+  test('when collapsed and the viewport changes from mobile to desktop and is expanded, scrolling is enabled', async function (assert) {
+    this.mockMedia();
+
+    let calls = [];
+    this.setProperties({
+      onDesktopViewportChange: (...args) => calls.push(args),
+    });
+
+    await render(hbs`<Hds::AppSideNav
+  @isCollapsible={{true}}
+  @onDesktopViewportChange={{this.onDesktopViewportChange}}
+>
+  <span id='test-app-side-nav-body' />
+</Hds::AppSideNav>`);
+    await this.changeBrowserSize(false);
+    assert.deepEqual(
+      calls[1],
+      [false],
+      'resizing to mobile triggers a false event'
+    );
+
+    await click('.hds-app-side-nav__toggle-button');
+
+    assert.dom('body', document).hasStyle({
+      overflow: 'hidden',
+    });
+
+    await this.changeBrowserSize(true);
+    assert.deepEqual(
+      calls[2],
+      [true],
+      'resizing to desktop triggers a true event'
+    );
+
+    assert.dom('body', document).doesNotHaveStyle('overflow');
+  });
+
+  test('when collapsed, the content in the AppSideNav is not focusable', async function (assert) {
+    await render(hbs`<Hds::AppSideNav
+    id='test-app-side-nav'
+  @isCollapsible={{true}}
+  @onDesktopViewportChange={{this.onDesktopViewportChange}}
+>
+  <span id='test-app-side-nav-body' />
```

Comment:
[question] what is this `span#test-app-side-nav-body` used for? 🤔

(I see other places where is not used in the tests assertions/declarations; maybe there's a reason for that? otherwise maybe we can remove them?)

Reply:
I can remove them... They were initially used in tests along with `<span id='hds-app-side-nav-hide-when-minimized' />` to show that inert was only added to the correct elements.